F21/joelanzi/_toLowerCase.py

```diff
@@ -0,0 +1,46 @@
+import pandas as pd
+from unicodedata import normalize
+
+## To Lowercase Function
+def _to_lower(word, language):
```

Comment:
This script could be structured differently to allow better code comprehension and reuse. For instance, we could add a utility file with classes or methods used for the lowercase tasks and separate it from the main code used for running the script.